src/content/en/fundamentals/getting-started/service-worker/before-we-start.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: shared/narrow
+title: "Before We Start"
```

Comment:
Can we re-title this something like "Service worker prerequisites" We should also call out other browser versions here, not just Chrome.